src/assets/custom/css/_sass/_announcement-slice.scss

```diff
@@ -1,42 +1,45 @@
 .announcement-slice {
-  background-color: $coral;
+  background-image: url("#{$site-base-url}/assets/custom/img/services/software-modernisation/bg-announcement-slice-softmod.jpg");
   color: white;
-  line-height: 34px;
-  padding-bottom: 10px;
-  padding-top: 10px;
+  padding-bottom: 15px;
+  padding-top: 15px;
   position: absolute;
   width: 100%;
-  z-index: 5;
+  z-index: 9999;
+  background-size: cover;
+  line-height: 1.5em;
+  text-align: center;
+
+  @media (min-width: 580px) {
+    line-height: 34px;
+
+    a {
+      margin-left: 5px;
+    }
+  }
 }
 
 .announcement-slice__inner {
-  @include max-width;
   padding-left: 15px;
   padding-right: 15px;
+  display: inline-block;
 
-  @media (min-width: 576px) {
-    max-width: 540px;
-    padding-left: 0;
-    padding-right: 0;
-  }
-  @media (min-width: 768px) {
-    max-width: 720px;
-  }
-  @media (min-width: 992px) {
-    max-width: 960px;
-    padding-left: 15px;
-    padding-right: 15px;
-  }
-  @media (min-width: 1200px) {
-    max-width: 1140px;
+  @media (min-width: 1024px) {
+    padding-left: 50px;
+    padding-right: 50px;   
   }
 }
 
 .announcement-slice__icon {
+  display: none;
   fill: transparent;
   float: left;
   margin-right: 15px;
   stroke-width: 5px;
   stroke: white;
   width: 30px;
-}
+
+  @media (min-width: 580px) {
```

Comment:
Same comment as above, regarding media queries and breakpoints

Reply:
Resolved here: https://github.com/codurance/site/pull/1924/commits/6c05d36e4190cd15f9dbfb11a569716e1bce7914